projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.html

```diff
@@ -108,7 +108,7 @@ <h1 class="mb-2">
                 *ngFor="let item of this.receivedItems"
                 [ngClass]="{'deleted-status': item.metadata.issue.status == 'deleted'}">
               <div class="col-sm-4 text">
-                <admin-record-masked *ngIf="item.metadata._masked" [record]="holding"></admin-record-masked>
+                <admin-record-masked *ngIf="item.metadata._masked" [record]="item"></admin-record-masked>
```

Comment:
Is it a fix? Maybe you can add an entry in the commit message.

Reply:
yes, this is the main fix about the related issue #2240.